README.md

```diff
@@ -27,6 +27,11 @@ ongoing on adding support for additional platforms. The installation
 process is documented in https://zulip.org/server.html and in more
 detail in [README.prod.md](README.prod.md).
 
+Running Zulip in Docker
+===========================
+
+Zulip in Docker. The installation process is documented in [README.docker.md](README.docker.md).
+
```

Comment:
Can you add an "(experimental)" note here?  And also move this doc to be a subsection of README.prod.md instead of in the main README.md, since that's where we document installing Zulip for production?